examples/egress/README.md

```diff
@@ -0,0 +1,12 @@
+##### WIP #####
+
+This examples show how to create Egress Rules:
+
+1) Create a third party resource called egresspolicy, with the following command:
+
+``kubectl create -f egress-resouce-yaml``
```

Comment:
This is good for now so long as the controller is resilient to the resource not existing - it would be great if the controller had the ability to install this TPR definition itself!